Sources/SpeziLLMOpenAI/LLMOpenAITokenSaver.swift

```diff
@@ -130,3 +126,8 @@ public class LLMOpenAITokenSaver: Module, EnvironmentAccessible, DefaultInitiali
         }
     }
 }
+
+
+extension CredentialsTag {
+    static let openAIKey = CredentialsTag.genericPassword(forService: "openai.com")
```

Comment:
Probably should be public as we reference it in the Docs in a way that it can be used by other parts of the app?

Reply:
we only say _that_ we store it in the keychain, but IMO that's only pointing out that an implementation detail (the fact that we store it) is done in a safe manner.

the previous version of this (the `LLMOpenAIConstants.credentialsServer` property) wasn't public either.

Reply:
Fair point; let's leave it internal for now.